docs/csharp/tuples.md

```diff
@@ -385,15 +404,31 @@ the last name are returned.
 
 [!code-csharp[Deconstruct extension method](../../samples/snippets/csharp/tuples/tuples/program.cs#13A_DeconstructExtension "Deconstruct a class type using an extension method")]
 
-You should be very careful defining multiple `Deconstruct` methods in a 
+You should be careful defining multiple `Deconstruct` methods in a 
 class or a class hierarchy. Multiple `Deconstruct` methods that have the
 same number of `out` parameters can quickly cause ambiguities. Callers may
 not be able to easily call the desired `Deconstruct` method.
 
-In this example, there is minimal chance for an ambiguous call because the 
+In this example, there is minimal chance for an ambiguous call because the
 `Deconstruct` method for `Person` has two output parameters, and the `Deconstruct`
 method for `Student` has three.
 
+Tuple equality enables more concise code for your types. The following example adds equality tests, and related overrides for `Person` class:
+
+[!code-csharp[EqualityWithTupleTests](../../samples/snippets/csharp/tuples/tuples/person.cs#SnippetEqualityTests "Equality tests for a Person type")]
+
+You construct a tuple for the members that participate in value-based equality, then test the tuples for equality.
+
+Deconstruction conversions are not applicable for testing equality. The following example does not compile:
```

Comment:
From the language perspective, deconstructions are carefully discussed as not being "conversions", because that would open questions of whether they should combine, or be inferred, etc.
Maybe "Deconstruction operators do not participate in testing equality"?